docs/differences-from-v1.md

```diff
@@ -81,3 +221,29 @@ If you are integrating a private project and want to share more details, or if y
 In 1.x, `fossa report` supported `fossa report dependencies`, `fossa report licenses` and `fossa report attribution`.
 
 In 2.x, only `fossa report attribution --json` is reported. The information in the previously supported v1 commands is contained in the output from `attribution`.
+
+## Frequently Asked Questions
+
+#### Until when CLI v1 will be supported by FOSSA?
+
+FOSSA 1.x CLI is available and can be used indefinitely. 
```

Comment:
Maybe mention something about us planning to modify the `install.sh` script 6 months (Does that seem right?). I would like people reading this to have at least some amount of pressure to upgrade to CLI v2.